frontend/app/templates/project/show/transportation/existing-conditions/study-area.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="sixteen wide column">
```

Comment:
Was there a reason you removed the wrapping row divs around these inner components? I think the table looked better with a little space between it and the map. But nitpicky design decisions im so not qualified to make :)

Reply:
I like this nitpick! Thanks for catching that and holding me to it. I actually shouldn't have removed the rows in this inner component, and at the parent `existing-condtions` template placed the `{{outlet}}` outside the row. I removed it as a (bad) fix get gutters between the stepper and the map.